src/xcode/ENA/ENA/Source/Extensions/PDFDocument+EmbedImage.swift

```diff
@@ -37,10 +38,25 @@ extension PDFDocument {
 			
 			context.cgContext.restoreGState()
 			
-			let imageRect = CGRect(x: position.x, y: position.y, width: image.size.width, height: image.size.height) // `CGRect` for the image.
+			// `CGRect` for the image.
+			let imageRect = CGRect(x: position.x, y: position.y, width: image.size.width, height: image.size.height)
 			
 			// Draw your image onto the context.
 			image.draw(in: imageRect)
+			
+			// Set the font as per the font size provided
+			let font = UIFont.preferredFont(forTextStyle: .body).scaledFont(size: textSize, weight: .regular)
+
+			// Set the attributes of the text
+			guard let paragraphStyle: NSParagraphStyle = NSParagraphStyle.default.mutableCopy() as? NSParagraphStyle else { return }
```

Comment:
Since the whole func is throws we could throw an error here instead of returning

Reply:
Removed paragraph style instead because it was not necessary. Handling throws in closures of system functions which are non throwing is tricky.